### ISSUE TITLE
[Improvement] Structures notifications

### ISSUE THREAD

**Issue by tigranm75** [eveseat/seat#568]
Hi.
Is it possible to add owner of structure in notifications (attacks and moon extractions)?
Thank you.

**Comment by tigranm75** [comment]
Also would be nice to add citadels low/full power notifications.
Thank you!

**Comment by Meganitrospeed** [comment]
Also would be nice to get the name of the structure in all cases specially with Low Fuel notifications

**Comment by LordMrcS** [comment]
To add on this, other kinds of notifications would be awesome. For instance when PI extractors reach it's end time.

**Comment by tigranm75** [comment]
And would be grate to add wardec notifications. Useful for altcorps - structures owners in highsec.
Thank you!

**Comment by warlof** [comment]
I got the feeling that's ticket is becoming a all in one marshmallows of notifications stuff.
If your request isn't related, can you please consider opening another issue? 😜

**Comment by sgeine** [comment]
@warlof there's another package that I believe was used by eve-intel that is freely available. perhaps since all the legwork has already been done that will give a significant jumpstart on this project as it would be tremendously helpful. https://github.com/flakas/reconbot

I don't see anyone clearly defining some specific features they'd like to see other than "structure notifications" I'll try to be more specific.
Moon pulls auto population would be spectacular with the existing calendar.
discord push notifications when a structure is:
1. attacked. this would be great as a ping.
2. when a structures fuel will expire along with 48 hours notice via a discord ping. this will require some role stuff as you wouldn't want everyone in the alliance knowing. perhaps making the information limited to a group.
3. a moon pull completes. this would be great if it can be made available to all both as a discord ping and in the calendar.
basically any of the push notifications you can find in the package I put up there would be fabulous. they can all be handled in different ways either as scheduled events populated in the seat calendar or as discord push pings.
